### PR TITLE
adjusted OMA to sync with future workflow

### PR DESCRIPTION
**Pull request by ndegunst**
As discussed with @antagomir yesterday, I updated OMA to include some more detailed explanation on different aspects of a TSE. I also updated the 3d plot in appendix A to a simpler plot using PCoA since it wasn't functioning.
With these adjustments, the future introductory workflow will not overlap with content in OMA.

### CODE REVIEW

File: inst/pages/97_extra_materials.qmd [hunk 1]
```diff
@@ -264,29 +264,33 @@ tse <- sampleMetadata %>%
...
-# Retrieving the explained variance
-e <- attr(reducedDim(tse_Genus, "PCoA_BC"), "eig");
-var_explained <- e/sum(e[e>0])*100
+tse_Genus <- runMDS(tse_Genus,
+              FUN = vegan::vegdist,
+              method = "bray",
+              ncomponents = 3,
+              assay.type = "relative_abundance",
+              name = "MDS_bray")
+
```
Comment (antagomir) [inline inst/pages/97_extra_materials.qmd:275]: Add explanation what the following rows do

File: inst/pages/97_extra_materials.qmd [hunk 2]
```diff
@@ -264,29 +264,33 @@ tse <- sampleMetadata %>%
...
-colData(tse_Genus)$most_abundant <- as.character(most_abundant)
-
-# Calculating percentage of the most abundant
-most_abundant_freq <- table(as.character(most_abundant))
-most_abundant_percent <- round(most_abundant_freq/sum(most_abundant_freq)*100, 1)
-
-# Retrieving the explained variance
-e <- attr(reducedDim(tse_Genus, "PCoA_BC"), "eig");
-var_explained <- e/sum(e[e>0])*100
+tse_Genus <- runMDS(tse_Genus,
```
Comment (antagomir) [inline inst/pages/97_extra_materials.qmd]: Add explanation that this adds new reducedDim data in the tse_Genus object

File: inst/pages/04_containers.qmd [hunk 3]
```diff
@@ -100,21 +108,26 @@ transformation. This is in fact, a requirement for the assays.
 
 ### colData
 
-`colData` contains data on the samples.
+colData contains information about the samples used in the study. This information can include details such as the sample ID, the primers used in the analysis, the barcodes associated with the sample (truncated or complete), the type of sample (e.g. soil, faecal, mock) and a description of the sample.
```
Comment (antagomir) [inline inst/pages/04_containers.qmd]: Perhaps better to put colData in quotes `colData` (here and elseheree for similar cases)

File: inst/pages/04_containers.qmd [hunk 4]
```diff
@@ -80,7 +84,11 @@ Individual assays can be accessed via `assay`
 assay(tse, "counts")[1:5,1:7]
 ```
 
-To illustrate the use of multiple assays, the relative abundance data can be 
+
+So, in summary, in the world of microbiome analysis, an assay is essentially a way to quantify and understand the composition of microbes in a given sample, which is super important for all kinds of research, from human health to the environment. 
+
+
+Further more, to illustrate the use of multiple assays, the relative abundance data can be
```
Comment (antagomir) [inline inst/pages/04_containers.qmd]: should be "Furthermore" (without space)

File: inst/pages/97_extra_materials.qmd [hunk 5]
```diff
@@ -264,29 +264,33 @@ tse <- sampleMetadata %>%
...
+e <- attr(reducedDim(tse_Genus, "MDS_bray"), "eig")
+rel_eig <- e / sum(e[e > 0])
+
+# Extract the coordinates as a dataframe
+coordinates <- as.data.frame(reducedDim(tse_Genus,"MDS_bray"))
+
+head(coordinates)
+
+# plot the coordinates using plotly and add the explained variance to the axes
+plot_ly(coordinates, x = ~V1, y = ~V2, z = ~V3, color = ~V3) %>%
```
Comment (antagomir) [inline inst/pages/97_extra_materials.qmd:285]: Is this under a subtitle on 3D plotting

**Review by antagomir — COMMENTED** [review]
Looks good! See few comments.

File: inst/pages/97_extra_materials.qmd (hunk 5 above)
Comment (ndegunst) [reply inst/pages/97_extra_materials.qmd:285]: Yes it is.

File: inst/pages/04_containers.qmd [hunk 6]
```diff
@@ -100,21 +108,26 @@ transformation. This is in fact, a requirement for the assays.
...
 colData(tse)
 ```
 
+To illustrate, ***X.SampleID*** gives the sample identifier, ***SampleType*** indicates the sample type (e.g. soil, fecal matter, control) and ***Description*** provides an additional description of the sample.
+
 ### rowData
 
 `rowData` contains data on the features of the analyzed samples. Of particular
 interest to the microbiome field, this is used to store taxonomic information.
+This taxonomic information is extremely important for understanding the composition and diversity of the microbiome in each sample analyzed. It enables you to identify the different types of micro-organisms present in samples, and to explore the relationships between microbiome composition and various environmental or health factors.
```
Comment (TuomasBorman) [inline inst/pages/04_containers.qmd]: Split long lines in multiple lines (80 character max). It is easier to see the history of the text when the lines are shorter

File: inst/pages/04_containers.qmd [hunk 7]
```diff
@@ -100,21 +108,26 @@ transformation. This is in fact, a requirement for the assays.
...
 
 ```{r rowdata}
 rowData(tse)
 ```
 
+
+
```
Comment (TuomasBorman) [inline inst/pages/04_containers.qmd]: Remove extra lines

**Comment by ndegunst** [comment]
should all be good now

File: inst/pages/97_extra_materials.qmd (hunk 5 above)
Comment (antagomir) [reply inst/pages/97_extra_materials.qmd:285]: resolve conversations when done.

**Comment by antagomir** [comment]
Ok - to merge. @TuomasBorman ?

**Comment by TuomasBorman** [comment]
Looks good!

**Pull request merged**